packages/dds/tree/CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # @fluidframework/tree
 
+## 2.0.0-rc.3.0.0
+
+### Major Changes
+
+-   Packages now use package.json "exports" and require modern module resolution [bef6aac1bc](https://github.com/microsoft/FluidFramework/commits/bef6aac1bc6a7d3c0f3dc9cbd53e4294c40b2720)
+
+    Fluid Framework packages have been updated to use the [package.json "exports"
+    field](https://nodejs.org/docs/latest-v18.x/api/packages.html#exports) to define explicit entry points for both
+    TypeScript types and implementation code.
+
+    This means that using Fluid Framework packages require the following TypeScript settings in tsconfig.json:
+
+    -   `"moduleResolution": "Node16"` with `"module": "Node16"`
+    -   `"moduleResolution": "Bundler"` with `"module": "ESNext"`
+
+    We recommend using Node16/Node16 unless absolutely necessary. That will produce transpiled JavaScript that is suitable
+    for use with modern versions of Node.js _and_ Bundlers.
+    [See the TypeScript documentation](https://www.typescriptlang.org/tsconfig#moduleResolution) for more information
+    regarding the module and moduleResolution options.
+
+    **Node10 moduleResolution is not supported; it does not support Fluid Framework's API structuring pattern that is used
+    to distinguish stable APIs from those that are in development.**
+
+### Minor Changes
+
+-   Better events [bef6aac1bc](https://github.com/microsoft/FluidFramework/commits/bef6aac1bc6a7d3c0f3dc9cbd53e4294c40b2720)
+
+    We have updated the Events to make it easier to create granular event listeners for single nodes and better support the
+    undo/redo feature. SharedTree nodes now expose `nodeChanged` and `treeChanged` events that fire in response to changes
+    in the node, and to changes in the subtree rooted at the node, respectively.
+
+    This change was originally made in [#20286](https://github.com/microsoft/FluidFramework/pull/20286) ([ac1e773960](https://github.com/microsoft/FluidFramework/commit/ac1e7739607551abb0dae7fa74dda56aec94b609)).
+
+    [Read more about SharedTree Events at fluidframework.com](https://fluidframework.com/docs/data-structures/tree/#event-handling)
+
+-   Recursive schemas [fa06668e34](https://github.com/microsoft/FluidFramework/commits/fa06668e340de9f7c2153c89ad33e37141f2fb50)
```

Comment:
the page we link isn't very useful. I don't see a way to even get to the commit message or PR from there. Looks like thats "Merge branch 'main' into rc3-release-notes" and "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

Reply:
Fixed in latest. I had to re-run generation after the release notes PR landed since that's where some of these
changesets were added. Some of the links are still going to be dumb and point to a seemingly random commit, but at least
that commit will exist.